src/test/java/com/sda/awesomemovies/api/awesomemoviesapi/movie/MovieRepositoryTest.java

```diff
@@ -0,0 +1,22 @@
+package com.sda.awesomemovies.api.awesomemoviesapi.movie;
+
+import com.sda.awesomemovies.api.movie.MovieEntity;
+import com.sda.awesomemovies.api.movie.MovieRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class MovieRepositoryTest {
+    @Autowired
+    private MovieRepository movieRepository;
+
+    @Test
+    public void addingNewMovies() {
+        movieRepository.save(new MovieEntity("Matrix"));
```

Comment:
Żeby to był test brakuje jeszcze jakiejś assercji, np wyciąganie tego filmnu z bazy i weryfikacja tytułu czy jest taki jak zapisywany. Możesz użyć tej metody, findByTitle żeby przetestować czy działa zgodnie  oczekiwaniami :-)

Reply:
poprawiono

Reply:
Nie widać zmian :-)

Reply:
tzn?

Reply:
```
        MovieEntity matrix = movieRepository.findByTitle("Matrix");
        String titleFromSql = matrix.getTitle();
        assertSame("Matrix", titleFromSql);
```

nie wiem dlaczego nie widac zmian skoro comitowalem

Reply:
Faktycznie, jest. Dokommituję zmiany w numerach migracji, które doszły do mastera w ostatnim czasie i zaraz to wmergujemy.